sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -124,9 +142,6 @@ class FileBrowserElement extends Polymer.Element {
   constructor() {
     super();
 
-    // TODO: Should choose the FileManager instance dynamically here based on
-    // the data source specified for this file-browser instance
-    this._fileManager = FileManagerFactory.getInstance();
     this._apiManager = ApiManagerFactory.getInstance();
```

Comment:
Let's move this one too to keep things consistent. If dependencies are built in the ready method let's do away with the constructor.